src/utils/render.js

```diff
@@ -0,0 +1,9 @@
+import { render } from 'pug'
+
+export default function renderDOM({ content, cb, query = '#root' }) {
+  const root = document.querySelector(query)
+  if (root) {
+    root.innerHTML = render(content)
+    if (cb) cb()
+  }
```

Comment:
Тоже можно сократить
```suggestion
  root?.innerHTML = render(content)
  cb?.()
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining